liquibase/changeLogs/capture/functions/pruneTimeSeriesData/pruneTimeSeriesData.sql

```diff
@@ -32,6 +32,8 @@ begin
 	execute format('create table if not exists %I.time_series_qualifiers_%s partition of %I.time_series_qualifiers for values in (%L)', '${AQTS_SCHEMA_NAME}', month_name, '${AQTS_SCHEMA_NAME}', month_number);
 	execute format('create table if not exists %I.field_visit_header_info_%s partition of %I.field_visit_header_info for values in (%L)', '${AQTS_SCHEMA_NAME}', month_name, '${AQTS_SCHEMA_NAME}', month_number);
 	execute format('create table if not exists %I.field_visit_readings_%s partition of %I.field_visit_readings for values in (%L)', '${AQTS_SCHEMA_NAME}', month_name, '${AQTS_SCHEMA_NAME}', month_number);
+	execute format('create table if not exists %I.field_visit_readings_%s partition of %I.field_visit_readings_by_loc for values in (%L)', '${AQTS_SCHEMA_NAME}', month_name, '${AQTS_SCHEMA_NAME}', month_number);
+	execute format('create table if not exists %I.field_visit_readings_%s partition of %I.datum_converted_values for values in (%L)', '${AQTS_SCHEMA_NAME}', month_name, '${AQTS_SCHEMA_NAME}', month_number);
```

Comment:
I think here you'll want `field_visit_readings_by_loc` on line 35 rather than `field_visit_readings` and similarly on line 36 you'll want `datum_converted_values` rather than `field_visit_readings` just after the `if not exists` statements.

Reply:
Further up just after line 21, you'll want to drop the `field_visit_readings_by_loc` and `datum_converted_values` tables as well.